File: C/1013 - O Maior.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int A,B,C,maiorAB,maiorABC;
    //printf("numeros:\n");
    scanf("%d %d %d",&A,&B,&C);
    maiorAB = (A+B+abs(A-B))/2;
    maiorABC = (maiorAB+C+abs(maiorAB-C))/2;
    printf("%d eh o maior\n",maiorABC);
    return 0;
}

File: Python 3/1013 - O Maior.py
A,B,C = input().split()
A,B,C = int(A), int(B), int(C)

maiorAB = (A+B+abs(A-B))/2
maiorAB_C = (maiorAB+C+abs(maiorAB-C))/2

print(int(maiorAB_C),'eh o maior')

